src/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -418,26 +499,120 @@ namespace
             return StatusCode::LibHostInvalidArgs;
         }
     }
+
+    int get_property(
+        const context_handle instance,
+        const pal::char_t *key,
+        const pal::char_t **value)
+    {
+        if (instance == nullptr || key == nullptr)
+            return StatusCode::InvalidArgFailure;
+
+        hostpolicy_context_t *context = static_cast<hostpolicy_context_t*>(instance);
+        if (!context->coreclr_properties.try_get(key, value))
+            return StatusCode::HostPropertyNotFound;
+
+        return StatusCode::Success;
+    }
+
+    int set_property(
+        const context_handle instance,
```

Comment:
Nit: personally I find the name `instance` confusing. I'm thinking about this as the handle... so I would call it just `handle`... but I can learn to live with `instance`.